### PR TITLE
Added graphviz to java buildbox

### PR DESCRIPTION
**Pull request by Tetradeus**
Will be used for schema spy for instance

### CODE REVIEW

File: java/Dockerfile.tpl [hunk 1]
```diff
@@ -44,6 +44,22 @@ RUN echo "Starting ..." && \
...
+        libcdt5 \
+        libcgraph6 \
+        libgd3 \
+        libgvc6 \
+        libgvpr2 \
+        libxaw7 \
+        libxmu6 \
+        libxt6 \
+        libann0 && \
+    curl http://ftp.debian.org/debian/pool/main/g/graphviz/graphviz_2.38.0-17_amd64.deb --output graphviz.deb && \
```
Comment (mremi) [inline java/Dockerfile.tpl]: You should add an argument to manage the version

File: java/Dockerfile.tpl [hunk 2]
```diff
@@ -44,6 +44,22 @@ RUN echo "Starting ..." && \
...
+    dpkg -i graphviz.deb && \
+    rm -f graphviz.deb && \
+    echo "Done Install base!" && \
```
Comment (mremi) [inline java/Dockerfile.tpl]: `base` -> `graphviz`

File: java/Dockerfile.tpl (hunk 1 above)
Comment (mremi) [reply java/Dockerfile.tpl]: And maybe use the Debian's package?

Comment (Tetradeus) [reply java/Dockerfile.tpl]: Done

File: java/Dockerfile.tpl (hunk 2 above)
Comment (Tetradeus) [reply java/Dockerfile.tpl]: Done

**Pull request merged**